### PR TITLE
Enable JWT rotation

### PR DESCRIPTION
**Pull request by arturo-seijas**
Applicable spec: <link>
N/A

### Overview

Enable ROTATE_REFRESH_TOKENS for token rotation

### Rationale

N/A

### Module Changes

settings.py

### Checklist

- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: app/settings.py [hunk 1]
```diff
@@ -147,6 +147,10 @@
     ],
 }
 
+SIMPLE_JWT = {
+    "ROTATE_REFRESH_TOKENS": True,
+}
```
Comment (kot0dama) [inline app/settings.py:152]: Thank you!

Would it be possible to also add more configuration items like:
```
    "SIGNING_KEY": "some secret key, possibly autogenerated on deployment",
    "ACCESS_TOKEN_LIFETIME": datetime.timedelta(days=1),
    "REFRESH_TOKEN_LIFETIME": datetime.timedelta(days=7),
```

Comment (kot0dama) [reply app/settings.py:152]: Well, maybe nevermind the `SIGNING_KEY` as it defaults to the Django app `SECRET_KEY`.

**Review by kot0dama — APPROVED** [review]
LGTM thanks!

**Review by javierdelapuente — DISMISSED** [review]

**Review by yanksyoon — DISMISSED** [review]
LGTM!

**Review by javierdelapuente — APPROVED** [review]

**Review by yanksyoon — APPROVED** [review]

**Pull request merged**